.gitignore

```diff
@@ -4,3 +4,4 @@
 *.iml
 .DS_Store
 /build
+/.project
```

Comment:
As far as .gitignore file, that was added by my GitEye client
automatically.  I'll look into it.
On Mar 18, 2015 10:12 PM, "Josh Guilfoyle" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/facebook/stetho/pull/95#discussion_r26725303>:
>
> > @@ -4,3 +4,4 @@
> >  *.iml
> >  .DS_Store
> >  /build
> > +/.project
>
> What is this for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/stetho/pull/95/files#r26725303>.
>